src/grantRegistry.ts

```diff
@@ -31,7 +35,7 @@ export function handleGrantCreated(event: GrantCreated): void {
     entity.lastUpdatedBlockNumber = event.block.number
     entity.lastUpdatedTimestamp = event.block.timestamp
     entity.createdAtTimestamp =
-        entity.createdAtTimestamp.toString() !== "0" ? entity.createdAtTimestamp : event.block.timestamp
+        entity.createdAtTimestamp != new BigInt(0) ? entity.createdAtTimestamp : event.block.timestamp
```

Comment:
A few lines up, why do we set `entity.metaPtr = metaPtr.id`? It's telling me that `entity.metaPtr` is a string, but I'd expect to be of type `MetaPtr`. This potentially seems related to the error we saw here where you can see the `metaPtr` for the last item in the array is a string instead of an object. cc @thelostone-mc 

![image](https://user-images.githubusercontent.com/17163988/143054500-81f8e7b5-cfa2-4448-99f2-fbbe546e2262.png)

Reply:
This sets the relation - the `MetaPtrs` are stored as a distinct entity (so they can be queried separately) - setting the ID is how you make that association.

Try and run this query in the playground (https://thegraph.com/hosted-service/subgraph/dcgtc/dc-gitcoin-grants-rinkeby?selected=playground): 

```
{
  grants(first: 5) {
    id
    owner
    payee
    metaPtr {
      id
      pointer
      protocol
    }
  }
}
```

<img width="857" alt="Screenshot 2021-11-23 at 18 16 56" src="https://user-images.githubusercontent.com/5897836/143081510-4ec4a4e1-9332-4023-823e-46e4b0000f4c.png">

Reply:
This sets the relation - the `MetaPtrs` are stored as a distinct entity (so they can be queried separately) - setting the ID is how you make that association.

Try and run this query in the playground (https://thegraph.com/hosted-service/subgraph/dcgtc/dc-gitcoin-grants-rinkeby?selected=playground): 

```
{
  grants(first: 5) {
    id
    owner
    payee
    metaPtr {
      id
      pointer
      protocol
    }
  }
}
```

This is what I see in indexedDB now that `feat-timestamps` has been merged:

<img width="857" alt="Screenshot 2021-11-23 at 18 16 56" src="https://user-images.githubusercontent.com/5897836/143081510-4ec4a4e1-9332-4023-823e-46e4b0000f4c.png">

Reply:
Perfect, LGTM, thanks!